backend/load_test/script/config/config.yml

```diff
@@ -0,0 +1,18 @@
+config:
+  target: "http://172.17.0.2:3000"
```

Comment:
What's this IP address ?
Shouldn't this be in the .env file instead?

Reply:
this is the url address of the back, and yes this is a mistake as it is. Not in the scope of the PR, 
to go deeper in explanation, 
When we use docker compose to orchestrate the application, 
we will use something like this if we want.
networks:
  transcendence:
    name: transcendence
    driver: bridge
    ipam:
      driver: default
      config:
        - subnet: 172.17.0.0/16
          gateway: 172.17.0.1